crates/python_extractor/src/lib.rs

```diff
@@ -1,18 +1,74 @@
+use anyhow::{Context, Result};
+use bzl_gen_build_python_utilities::PythonProgram;
+use bzl_gen_build_shared_types::api::extracted_data::{DataBlock, ExtractedData};
+use encoding_rs::*;
 use std::{
-    collections::{BTreeSet, HashSet},
-    io::Read,
+    collections::{BTreeSet, HashMap, HashSet},
+    io::{BufRead, Read},
     path::PathBuf,
 };
 
-use bzl_gen_build_python_utilities::PythonProgram;
-
-use bzl_gen_build_shared_types::api::extracted_data::{DataBlock, ExtractedData};
-
-use anyhow::{Context, Result};
-
 mod extract_py_bzl_gen_build_commands;
 mod extract_py_imports;
 
+fn get_codec_map() -> HashMap<&'static str, &'static Encoding> {
```

Comment:
can this be `const`?

I asked gpt and it said something like this would work:
```rust
use std::collections::HashMap;
use lazy_static::lazy_static;

lazy_static! {
    static ref MY_MAP: HashMap<&'static str, &'static str> = {
        let mut m = HashMap::new();
        m.insert("key1", "value1");
        m.insert("key2", "value2");
        m
    };  
}

```